lib/collector_registration.rb

```diff
@@ -29,6 +32,39 @@ def configure_uc6
     end
   end
 
+  def configure_vsphere
+    begin
+      return false unless can_connect?(:vsphere_host, 443)
+      if ( @configuration.vsphere_configured? )
+        Timeout::timeout(15) do
+          VSphere::refresh
+          session = VSphere::session
+          if ( session.serviceInstance.content.rootFolder )
+             if ( MetricsCollector::level_3_statistics_enabled? )
+	       logger.info "Succesful connected to vsphere"
```

Comment:
I would use single quotes here since there is no variable interpolation within the string